src/jump_moi_overloads.jl

```diff
@@ -80,6 +80,42 @@ function MOI.get(
     return _moi_get_result(JuMP.backend(model), attr, JuMP.index(var_ref))
 end
 
+# extras to handle model_dirty
+
+function MOI.get(
+    model::JuMP.Model,
+    attr::ReverseConstraintSet,
+    var_ref::JuMP.ConstraintRef,
+)
+    JuMP.check_belongs_to_model(var_ref, model)
+    return _moi_get_result(JuMP.backend(model), attr, JuMP.index(var_ref))
+end
+
+function MOI.set(
+    model::JuMP.Model,
+    attr::ForwardConstraintSet,
+    con_ref::JuMP.ConstraintRef,
+    set::MOI.AbstractScalarSet,
+)
+    JuMP.check_belongs_to_model(con_ref, model)
```

Comment:
I just got confused about this and was wondering why we need to redefine these and double checked whether we already had a fallback for this in JuMP. We should have a clear header somewhere explaining the issue with `model_dirty` and we should refer to this in this function

Reply:
we had a `# FIXME` in the code and now I added more details